content/psy_recommend.md

```diff
@@ -0,0 +1,7 @@
+title: Психологи рекомендут
+slug: psy_recommend
+tags: [_nav, _ignore_nav]
+type: no_self_content
+---
+
+empty
```

Comment:
I don't think it's informative or really that necessary. Remove the last two lines.